mediapicker/src/main/java/iam/thevoid/mediapicker/builder/VideoIntentBuilder.java

```diff
@@ -25,8 +25,8 @@ public VideoIntentBuilder setVideoFileSize(long videoFileSize, @NonNull SizeUnit
         return this;
     }
 
-    public VideoIntentBuilder setVideoQuality(VideoQuality videoQuality) {
-        this.videoQuality = videoQuality.getQuality();
```

Comment:
Ok, but. What means "long videoQuality"? I think first of all here must be int instead of long, secondly we can add IntDef annotation with named constants, it is simper to read and understand

Reply:
It's long because the other properties as well, but it can be an int, of course